### DISCUSSION TITLE
Redirect root domain to website

### DISCUSSION THREAD

**Discussion by mattsoseman** [microsoft/AzUrlShortener#496] [General]
Hi - How can I redirect the root domain (e.g. contoso.com) to my company's website (e.g. www.fabrikam.com)? Currently the root domain goes to this page:
![image](https://github.com/microsoft/AzUrlShortener/assets/136049104/b2cfca5c-1596-4e50-9b2b-d7779d470a99)

**Comment by fboucher** [comment]
That could be an idea @mattsoseman. When no vanity is passed, we could redirect to a pre-defined URL. Make a feature request.

**Comment by mattsoseman** [comment]
How do I make a feature request?

-Matt
________________________________

**Reply by fboucher** [reply]
@mattsoseman,  I did it for you here #514

**Comment by mattsoseman** [comment]
Thank you!!! Any ETA?

-Matt
________________________________

**Comment by gjbundy** [comment]
Could really use this too!
